### PR TITLE
-aws.0.1.0, +aws.1.0.0

### PR DESCRIPTION
**Pull request by seliopou**
Closes ocaml/opam-repository#5619

### CODE REVIEW

File: packages/aws/aws.1.0.0/opam [hunk 1]
```diff
@@ -1,12 +1,9 @@
 opam-version: "1.2"
-name: "aws"
-version: "0.1.0"
 maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
 authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>"
            "Daniel Patterson <dbp@dbpmail.net>" ]
 license: "BSD-3-clause"
 homepage: "https://github.com/inhabitedtype/ocaml-aws"
-dev-repo: "https://github.com/inhabitedtype/ocaml-aws.git"
```
Comment (dsheets) [inline packages/aws/aws.1.0.0/opam]: Why was this removed?

**Comment by seliopou** [comment]
Unintentional, force-pushed the fix.

**Comment by lefessan** [comment]
`tar ztf aws-v1.0.0.tar.gz` shows that they are multiple files at the root of the archive, instead of only one (should probably be `aws-v1.0.0`. Could you fix that ?

**Comment by seliopou** [comment]
Used the wrong version of the release script, I've updated the archive and force-pushed a change to the checksum.
